biojava-structure/src/main/java/org/biojava/nbio/structure/align/util/URLConnectionTools.java

```diff
@@ -119,72 +83,66 @@ public static HttpURLConnection openHttpURLConnection(URL url, int timeout)
 	 * @throws ConnectException
 	 *
 	 * */
-	public static HttpURLConnection openHttpURLConnection(URL url)
-	throws IOException, ConnectException {
+	public static URLConnection openURLConnection(URL url)
+			throws IOException, ConnectException {
```

Comment:
ConnectException is not actually thrown by anything... it shouldn't be declared in the `throws`